src/garage/torch/policies/deterministic_mlp_policy.py

```diff
@@ -63,6 +64,7 @@ def get_action(self, observation):
                         distribution
 
         """
+        observation = torch.Tensor(observation)
```

Comment:
i think it's fair for this primitive to assume that its input is a `torch.Tensor`. it even says so in the docstring.

is there a reason you can't do this upstream of this primitive?

Reply:
+1